### PR TITLE
link to 1mn video example for int. RSE day

### PR DESCRIPTION
**Pull request by annefou**
Added an example of a 1mn video (session 1 of our international RSE day). I hope it can encourage RSEs to post their video!

### CODE REVIEW

File: content/events/international-rse-day/_index.md [hunk 1]
```diff
@@ -17,13 +17,15 @@ The Nordic-RSE association is contributing to the International RSE day with an
...
      - your name, institution, RSE project you are working on e.g. what software are you contributing to (language, topic and whatever you can say in less than 1 minute!)
 
      All the videos will be published in our youtube channel and we will also make a "wall of fame" on the nordic-RSE website. 
 
-     Nordic-RSE staff will provide helpdesk so they can support you to prepare the video but of course, feel free to skip this session and send us directly the link to your 1 minute video!
+     Nordic-RSE staff will provide helpdesk in our [nordic-rse/international RSE day Oct 14 stream](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse/topic/international.20RSE.20day.20Oct.2014) so they can support you to prepare the video but of course, feel free to skip this session and send us directly the link to your 1 minute video!
+     
+     If you want to an example of what you can have in your 1mn video, have a look [here](https://vimeo.com/629975390). Once your video is ready, share the link with us on our [nordic-rse/international RSE day Oct 14 stream](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse/topic/international.20RSE.20day.20Oct.2014).
```
Comment (lucaferranti) [inline content/events/international-rse-day/_index.md:28]: When I clicked the vimeo link it said "video not rated, log in to watch", is this intentional? (I never used vimeo)

Comment (annefou) [reply content/events/international-rse-day/_index.md:28]: Weird. Could it depend on the browser? Even when using an incognito window, I don't get this message! Or maybe the video was not fully published yet and it is now?

Comment (lucaferranti) [reply content/events/international-rse-day/_index.md:28]: using incognito in chrome
image

Comment (annefou) [reply content/events/international-rse-day/_index.md:28]: Ok. I got it. Is it better now? I did not know I had to se the audience manually! Thanks!.

Comment (lucaferranti) [inline content/events/international-rse-day/_index.md:28]:
```suggestion
     If you want to see an example of what you can have in your 1mn video, have a look [here](https://vimeo.com/629975390). Once your video is ready, share the link with us on our [nordic-rse/international RSE day Oct 14 stream](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse/topic/international.20RSE.20day.20Oct.2014).
```

**Review by lucaferranti — APPROVED** [review]

**Pull request merged**